dont_clone_monorepo.py

```diff
@@ -0,0 +1,12 @@
+from talon import app
+
+
+def on_ready():
+    app.notify(
+        "Don't clone Cursorless monorepo",
+        "For Talon use cursorless-talon instead:",
+        "https://github.com/cursorless-dev/cursorless-talon",
```

Comment:
```suggestion
        "Whoops! You cloned our monorepo instead of cursorless-talon",
        "Please remove cursorless and clone cursorless-talon instead:",
        "https://github.com/cursorless-dev/cursorless-talon",
```